### ISSUE TITLE
Specify AFP Device Type

### ISSUE THREAD

**Issue by jitteryjuice** [cilix-lab/docker-netatalk#1]
Hey,

Great work on this! Are you able to provide an example if using docker-compose?
Another question, is it possible to implement specifying what time of device is "hosting" the server? aka, Time Capsule etc

Cheers

**Comment by cilix-lab** [comment]
Hi!

Normally I'd use this container in a host with avahi mDNS running on it, so I can publish the container's port without having it to be on the host's network. On a regular Ubuntu setup with avahi installed on it, a file /etc/avahi/services/afp.service, containing something like:

`<?xml version="1.0" standalone="no"?>`
`<!DOCTYPE service-group SYSTEM "avahi-service.dtd">`
`<service-group>`
`<name replace-wildcards="yes">%h</name>`
`<service>`
`<type>_afpovertcp._tcp</type>`
`<port>548</port>`
`</service>`
`<service>`
`<type>_device-info._tcp</type>`
`<port>0</port>`
`<txt-record>model=RackMac</txt-record>`
`</service>`
`</service-group>`

... should do the trick. It will publish the AFP service and having the _device-info._tcp tags will allow you to choose which type of device you wan't Macs to recognize. You can use any other identifier in the txt-record tag.

About the docker-compose part, what would you like to accomplish? I guess you could use this container and an avahi one, so as to  link the services together with docker-compose, but you would have to open the avahi container to the host's network. I'll look into it and try to come up with an example.

**Comment by jitteryjuice** [comment]
Hi,

Thanks for the reply! I suppose I was hoping to be able to specify the
device type as an environment field so that it is easily identified instead
of editing the container directly, but I have edited and know how to edit
the avahi file :).

I've noticed it doesn't show up on my Macs finder, only when I connect
manually.. though my debian should auto port forward the necessary ports
for avahi.

Thanks again for the container, works great !
